test/ably/restchannelhistory_test.py

```diff
@@ -217,10 +208,8 @@ def test_channel_history_time_backwards(self):
             history0.publish('history%d' % i, i)
             time.sleep(0.1)
 
-        time.sleep(16)
-
-        history = history0.history(direction='backwards', start=interval_start,
-                end=interval_end)
+        history = history0.history(SearchParams(backwards=True, limit=25, start=interval_start,
```

Comment:
Why has the API changed so that it needs a SearchParams class?  This does not match most of our other APIs, we have tried to use named arguments where possible.